libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/NavigationInstructionPlayer.java

```diff
@@ -8,27 +8,23 @@
 
 import com.mapbox.services.android.navigation.v5.milestone.VoiceInstructionMilestone;
 
-import java.util.Queue;
-import java.util.concurrent.ConcurrentLinkedQueue;
-
 public class NavigationInstructionPlayer implements InstructionListener {
 
   private AudioManager instructionAudioManager;
   private AudioFocusRequest instructionFocusRequest;
   private MapboxSpeechPlayer mapboxSpeechPlayer;
   private AndroidSpeechPlayer androidSpeechPlayer;
-  private Queue<VoiceInstructionMilestone> instructionQueue;
+  VoiceInstructionMilestone voiceInstructionMilestone;
```

Comment:
Does it make sense to store the `String` `voiceInstructionMilestone.getAnnouncement()` here instead of the milestone?  Also this field can be `private` rather than package-private.

Reply:
@danesfeder no because we need both the `announcement` and the `ssmlAnnouncement`

Reply:
Ah okay, wasn't sure because `ssmlAnnouncement` is immediately passed to:
`mapboxSpeechPlayer.play(voiceInstructionMilestone.getSsmlAnnouncement());`